build-cran-package.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+# [description]
+#     Prepare a source distribution of the R package
+#     to be submitted to CRAN.
+#
+# [usage]
+#     sh build-cran-package.sh
+
+set -e
+
+ORIG_WD=$(pwd)
+TEMP_R_DIR=$(pwd)/lightgbm_r
+
+if test -d ${TEMP_R_DIR}; then
+    rm -r ${TEMP_R_DIR}
+fi
+mkdir -p ${TEMP_R_DIR}
+
+ # move relevant files
+cp -R R-package/* ${TEMP_R_DIR}
+cp -R include ${TEMP_R_DIR}/src/
+cp -R src/* ${TEMP_R_DIR}/src/
+
+cd ${TEMP_R_DIR}
+
+    # recreate configure script if autoconf is available,
```

Comment:
Why indent here?..

Reply:
when I write shell scripts that changing the working directory, I like to indent over so I know that all that code in the indents is inside the new directory. Usually I use `pushd` and `popd` for that but those are not guaranteed to be there with `sh`

Reply:
Ah, I see! I thought that it was just outdated artifact after removed some `if` statement or something like this 🙂